tests/native_tests/with_test.hy

```diff
@@ -34,12 +34,37 @@
         (assert (= t2 2))
         (assert (= t3 3))))
 
-  (defn test-quince-with []
-    "NATIVE: test four withs, one with no args"
-    (with [t1 (WithTest 1)
-           t2 (WithTest 2)
-           t3 (WithTest 3)
-           _ (WithTest 4)]
-          (assert (= t1 1))
-          (assert (= t2 2))
-          (assert (= t3 3))))
+(defn test-quince-with []
+  "NATIVE: test four withs, one with no args"
+  (with [t1 (WithTest 1)
+         t2 (WithTest 2)
+         t3 (WithTest 3)
+         _ (WithTest 4)]
+        (assert (= t1 1))
+        (assert (= t2 2))
+        (assert (= t3 3))))
+
+(defclass SuppressZDE [object]
+  (defn --enter-- [self])
+  (defn --exit-- [self exc-type exc-value traceback]
+    (and (not (none? exc-type)) (issubclass exc-type ZeroDivisionError))))
+
+(defn test-exception-suppressing-with []
+  ; https://github.com/hylang/hy/issues/1320
```

Comment:
If I make the editor indent automatically, then I have to wrestle with it when it's making a bad decision.

> Lisp must be indented correctly to be legible. That particular rule is absolute.

I think this code is legible (or at least, it was when I started). I know that's inherently subjective, but you and the other two Horsemen of Hy Maintenance have approved my PRs in the past, so I don't see what's changed. I'm not asking for new standards in reviewing PRs; I'm asking for what I'm used to.

Anyway, if things like the number of semicolons for a comment, or whether the indent lines up with the immediate parent form instead of an ancestor form without other hanging children, are going to be a blocker for you from now on, then I guess you're not going to approve my PRs. It's a shame, since you're whip-smart and notice genuinely important things that others miss, but enforcing style rules for their own sake is counterproductive.

I think [the review process here](https://github.com/hylang/hy/pull/1377) is a good model. The author indented something in a nonstandard way (for Python and our codebase), I pointed it out, the author said "I prefer it that way", I said "Eh, no biggie" and @kirbyfan64 said "it's not really a blocking issue" and merged it.